### PR TITLE
Add ragel and a52

### PR DESCRIPTION
**Pull request by superloach**
Working toward #2785

### CODE REVIEW

**Comment by superloach** [comment]
Forgot how branches work 🤦‍♀️
These 2 can just go together, haha

File: packages/ragel.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'package'
+
+class Ragel < Package
+  description 'ragel desc stub'
```
Comment (uberhacker) [inline packages/ragel.rb]: Can we change this description?  Does `Ragel State Machine Compiler` make it more clear?

File: packages/ragel.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  homepage 'http://www.colm.net/open-source/ragel/'
+  version '6.10'
+  source_url 'http://www.colm.net/files/ragel/ragel-6.10.tar.gz'
```
Comment (uberhacker) [inline packages/ragel.rb]: `https://www.colm.net/files/ragel/ragel-6.10.tar.gz` works so can we use that instead?

File: packages/ragel.rb (hunk 1 above)
Comment (superloach) [reply packages/ragel.rb]: Dang it, forgot to change that. Fixing now.

File: packages/ragel.rb (hunk 2 above)
Comment (superloach) [reply packages/ragel.rb]: Yes!

**Comment by uberhacker** [comment]
Nice work @superloach!

**Pull request merged**